javers-spring-boot-starter-mongo/src/main/java/org/javers/spring/boot/mongo/JaversMongoAutoConfiguration.java

```diff
@@ -50,7 +49,8 @@ public Javers javers() {
 
         logger.info("connecting to database: {}", mongoProperties.getMongoClientDatabase());
 
-        JaversRepository javersRepository = new MongoRepository(mongoDatabase);
+        MongoRepository javersRepository = new MongoRepository(mongoDatabase);
```

Comment:
please move this flag to the new constructor & factory method:

```
MongoRepository DocumentDBRepository = MongoRepository.createDocumentDBRepository(mongoDatabase);
```

(we don't use setters)